_posts/2020-12-02-spack-user-survey-2020.md

```diff
@@ -0,0 +1,918 @@
+  </a>
+  <figcaption>
+    Feature ratings by workplace (left) and by job type (right).
+  </figcaption>
+</figure>
+
+In addition to the community-wide averages above, we looked at whether
+different segments of the community rated features differently. On the
+right above, we split out average feature ratings by workplace, and on
+the left we split them out by job type.
+
+Overall, the rank order of features was similar across different
+workplaces and job types. Reusing existing installs and the new
+concretizer were consistently at the top of everyone's list, and the
+lowest-rated features had low ratings across the board. Industry users
+prioritized cloud integration noticeably higher than other groups, and
+user support staff placed a much higher value on package testing than
+other job types (perhaps because they are involved in more package
+testing efforts at their sites). Managers and ASCR labs rated separate
+build dependencies lower than other groups. Other than these outliers
+there were not significant deviations from the overall order of
+preference.
+
+There are some noticeable trends *across* groups. System administrators,
+user support staff, and industry users tended to rate features as less
+important across the board. It's hard to know how to interpret this -- it
+could mean that they're happy with the existing capabilities of Spack, or
+that these particular improvements aren't their top priorities.
+
+## If we had a (virtual) workshop on Spack, would you attend?
+
+We've thought about having a Spack user meeting for a while, and we had
+actually started planning for an inaugural Spack User Meeting earlier this
+year. That fell apart when the pandemic hit. Other similar tools have had
+good luck with meetings like this (e.g.,
+[NixCon](https://2020.nixcon.org/) and the
+[EasyBuild User Meeting](https://github.com/easybuilders/easybuild/wiki/5th-EasyBuild-User-Meeting)),
+so we asked users what they thought of a potentially virtual meeting:
+
+<figure>
+  <a href="/assets/images/spack-user-survey-2020/two_pies_would_attend_workshop.svg">
+    <img src="/assets/images/spack-user-survey-2020/two_pies_would_attend_workshop.svg">
+  </a>
+</figure>
+
+Just over half of users (over 85 people) said they would attend, and 8
+said they'd be willing to present on their work. That seems like more
+than enough for an initial Spack meeting, so expect us to announce
+something for 2021.
+
+# Getting Help
+
+We're interested in making it easier to learn about Spack, so we asked
+people how they're doing it now.
+
+## Have you done a Spack tutorial?
+
+<figure>
+  <a href="/assets/images/spack-user-survey-2020/two_pies_did_tutorial.svg">
+    <img src="/assets/images/spack-user-survey-2020/two_pies_did_tutorial.svg">
+  </a>
+</figure>
+
+We were surprised to see that over 60% of all our users have done a Spack
+tutorial. Since 2016, We've been doing Spack tutorials at conferences
```

Comment:
```suggestion
We were surprised to see that over 60% of all our users have done a [Spack
tutorial](https://spack-tutorial.readthedocs.io/en/latest/). Since 2016, We've been doing Spack tutorials at conferences
```